common/traits/wc_religion_traits.txt

```diff
@@ -0,0 +1,25 @@
+﻿scarlet_mark = {
+	category = fame
+	monthly_piety_gain_mult = -0.8
+	zealot_opinion = -70
+	
+	shown_in_ruler_designer = no
+
+	triggered_opinion = {
+		parameter = scarlet_mark_active
+		opinion_modifier = tenet_scarlet_mark_crime
+		same_faith = yes
```

Comment:
I don't know if "faith = faith:scarlet" can be done
Can it ?

Reply:
You can just remove the `same_faith = yes`. Since it has the parameter `scarlet_mark_active` it will only activate on religions that care about scarlet mark (only the scarlets.)

Reply:
done